kolibri/core/templatetags/core_tags.py

```diff
@@ -71,3 +72,12 @@ def theme_favicon():
     favicon_url = favicon_urls[0] if favicon_urls else static("assets/logo.ico")
 
     return format_html('<link rel="shortcut icon" href="{}">', favicon_url)
+
+
+@register.simple_tag()
+def site_title():
+    """
+    Return the text of the site title, if provided by the theme. If not, the
+    default will be returned.
```

Comment:
Tiny suggestion: Might be worth mentioning that the site title is translatable, and this is mostly to allow for transliteration into other alphabets (given that sites often brand themselves with the same name across many languages, so it doesn’t need translating in that respect).

Suggested text:
```suggestion
    Return the text of the site title, if provided by the theme. If not, the
    default will be returned. The site title may be translated, to allow for
    transliteration into other alphabets where needed.
```